File: posneg.c
#include<stdio.h>
#include<conio.h>
void main()
{
int s;
printf("Input");
scanf("%d",&s);
if(s>=0){
printf("Positive");
}
if(s<=0){
printf("Negative");
}
if(s==0)
{
printf("Zero");
}
getch();
}

File: posneg.java
import java.util.Scanner;
public class num
{
public static void main(String[] args)
{
int number;
Scanner scan=new Scanner(System.in);
System.out.print("Input");
number= scan.nextInt();
scan.close();
if(number>0)
{
System.out.println("Positive");
}
else if(number<0)
{
System.out.println("Negative");
}
else{
System.out.println("Zero");
}
}
}
